### PR TITLE
Update BASE and LINEA support

### PR DESCRIPTION
**Pull request by corlard3y**
## Pull Request Template

#1636 

### Description

- **Problem/Feature**:
- So we are basically just adding Linea chain support, it’s just going to be a new chain like the others we have, from polygon, Arbitrum, cyber and the rest that have been added beforehand. So this is a new one called Linea Chain - So you can create channels, send notifications on this chain.
- 
- Uncomment Base Chain changes

### Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Code refactor
- [ ] Documentation update
- [ ] Other (please describe):

### Checklist

- [ ] **Quick PR**: Is this a quick PR? Can be approved before finishing a coffee.
  - [ ] Quick PR label added
- [ ] **Not Merge Ready**: Is this PR dependent on some other PR/tasks and not ready to be merged right now.
  - [ ] DO NOT Merge PR label added

### Frontend Guidelines

- [x] Followed frontend guidelines as per the [Guidelines Notion Doc](https://www.notion.so/pushprotocol/Frontend-dApp-Guidelines-1d7806ae3d9e4569a340b563dcd0536c)

### Build & Testing

- [ ] No errors in the build terminal
- [ ] Engineer has tested the changes on their local environment
- [ ] Engineer has tested the changes on deploy preview

### Screenshots/Video with Explanation

- **Before:** Explain the previous behavior

- **After:** What's changed now
Screenshot 2024-08-21 at 10 43 14

### Review & Approvals

- [ ] Self-review completed
- [ ] Code review by at least one other engineer
- [ ] Documentation updates if applicable

### CODE REVIEW

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]
config.alpha needs to be updated too

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]
image
 the base and linea icon in add new chain page arent of the correct size, it increases the select size

**Comment by corlard3y** [comment]
Fixed all comments @mishramonalisha76

**Review by mishramonalisha76 — APPROVED** [review]

**Review by abhishek-01k — APPROVED** [review]

**Review by rohitmalhotra1420 — CHANGES_REQUESTED** [review]

File: public/svg/Linea.svg
[the remark this answers is not in the document]
Comment (corlard3y) [reply public/svg/Linea.svg:1]: Yes, we do. The main Chain switcher component in the header uses that till we update it to use the new Icons block component from the root files across all chains. Check LOGO_FROM_CHAIN_ID in UtilityHelper @rohitmalhotra1420

**Review by rohitmalhotra1420 — APPROVED** [review]

**Pull request merged**